src/tests/forgotPassword.test.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { expect, describe, it } from 'vitest';
+import ForgotPassword from '../pages/ForgotPassword';
+
+import { render } from '../utilis/testUtilis';
+
+describe('Forgot Password Component', () => {
+	it('should render forgot password correctly', () => {
+		render(<ForgotPassword />);
+		const element = screen.getByRole('heading');
+		expect(element).toBeInTheDocument();
+	});
+
+	it('should display an email input field', () => {
+		render(<ForgotPassword />);
+		const input = screen.getByPlaceholderText('Email');
+		expect(input).toBeInTheDocument();
+	});
+
+	it('should display a Request Reset button', () => {
+		render(<ForgotPassword />);
+		const button = screen.getByRole('button');
+		expect(button).toBeInTheDocument();
+	});
+
+	it('should display a "back to login" link', () => {
```

Comment:
- Add more test scenarios other than component render and element toBeInTheDocument.